readme.md

```diff
@@ -74,7 +74,15 @@ docker run -d \
   -v </path/to/config>:/config \
   -v </path/to/metadata>:/metadata \
   --name audiobookshelf \
-  --rm ghcr.io/advplyr/audiobookshelf
+  ghcr.io/advplyr/audiobookshelf
+```
+
+### Docker Update
+
+```bash
+docker stop audiobookshelf
+docker pull advplyr/audiobookshelf
```

Comment:
This should be the new ghcr.io registry

Reply:
I have updated the repo to point to the new ghcr.io registry.